### DISCUSSION TITLE
Why is MySQL DB size too Big - 3 months of usage and the SQL DB size has become approx 20GB

### DISCUSSION THREAD

**Discussion by harshdoshihd** [YetiForceCompany/YetiForceCRM#13939] [General]
Hi There, 

We have implemented Yetiforce as a pilot and the SQL db size has gone up to 25GB in just 3 months of usage.

We are not uploading and pdf, images, docs, we are just working on typed data.

![yetiforce db](https://user-images.githubusercontent.com/61234565/102583607-03ea0680-412b-11eb-84bf-48571ff4b776.png)

Attached herewith is the screenshot of db files which are consuming lots of space. I think one of he file is related to history tracking, other i didnt understood. Can I get a documentation of what are these directories for and how do I reduce the disk usage.

**Comment by mariuszkrzaczkowski** [comment]
I see notifications or Track Changes are on, do you need this?

**Comment by harshdoshihd** [comment]
Hi Mariusz,

I am currently deleting all notifications. 

But for Track changes, I just need it for Contacts, precisely for phone numbers and emails - to keep a track if any employee purposely changes the data.

**Comment by mariuszkrzaczkowski** [comment]
Maybe you have something redundantly configured that you have 16 million notifications,
you can disable the history of changes for the Notifications module
and when you delete it, delete it permanently, not to the trash

**Reply by harshdoshihd** [reply]
Hi Marius,

Yes, I understood the mistake i made for notifications, I have rectified it.

Secondly, Where do I find an option to disable history of changes for the Notifications module I think this will help a lot to save db size?

Thanks in advance.

Best,
Harsh

**Reply by mariuszkrzaczkowski** [reply]
https://gitdeveloper.yetiforce.com/index.php?module=ModTracker&parent=Settings&view=List

**Reply by harshdoshihd** [reply]
Thanks a lot!

**Reply by harshdoshihd** [reply]
Hi Marius,

One more thing, now how do I delete old notification records from the db - do I need to manually delete it from db or is there any console from where I shall be able to select and delete them.

Best,
Harsh

**Reply by harshdoshihd** [reply]
Hi Marius,

I deleted data from notification table, but the size of the table has not reduced, I tried to optimize it from SQL, but its not allowed. May I know the path in Yetiforce from where I can optimize the tables.

Best,
Harsh

**Reply by mariuszkrzaczkowski** [reply]
It's not that simple, we have special scripts for cleaning garbage.
You have to know very well the structure of the database to be able to delete something

**Reply by vovpff** [reply]
@mariuszkrzaczkowski what script you mean?

**Reply by mariuszkrzaczkowski** [reply]
PHP script for garbage removal

**Reply by vovpff** [reply]
@mariuszkrzaczkowski thx. Is this script shared somewhere?

**Reply by mariuszkrzaczkowski** [reply]
No, this is our internal script

**Reply by vovpff** [reply]
@mariuszkrzaczkowski ok. This is sad. Maybe you can share it?

**Reply by mariuszkrzaczkowski** [reply]
We will think about it, talk it over internally

**Reply by harshdoshihd** [reply]
Do let us know about it! & Thanks for the support.
